example/MODULE.bazel

```diff
@@ -8,7 +8,7 @@ bazel_dep(name = "rules_buf", version = "0.2.0")
 bazel_dep(name = "bazel_skylib", version = "1.4.2")
 bazel_dep(name = "rules_java", version = "5.5.0")
 bazel_dep(name = "rules_jvm_external", version = "4.5")
-bazel_dep(name = "rules_go", version = "0.42.0")
+bazel_dep(name = "rules_go", version = "0.42.0", repo_name = "io_bazel_rules_go")
```

Comment:
I guess this alias name is unavoidable for some reason?

Reply:
I was getting errors, yes. Not sure it is avoidable.

Reply:
okay I'll take a stab at removing it in a followup, maybe. It does feel like the name we choose here leaks into userspace, at least for WORKSPACE users.